store/sqlstore/channel_store.go

```diff
@@ -521,9 +481,16 @@ func (s SqlChannelStore) createInitialSidebarCategoriesT(transaction *gorp.Trans
 	}
 
 	if !hasCategoryOfType(model.SidebarCategoryFavorites) {
+		favoritesCategoryId := model.NewId()
+
+		// Create the SidebarChannels first since there's more opportunity for something to fail heree
```

Comment:
```suggestion
 		// Create the SidebarChannels first since there's more opportunity for something to fail here
```

Reply:
Ugh, I need to clean my keyboard or something. I make typos like that constantly.